### PR TITLE
adding nodejs week1 homework

### PR DESCRIPTION
**Pull request by varsha84**

### CODE REVIEW

File: nodejs/week1/homework/src/backend/app.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  response.json(reservations);
+});
+
+app.get("/reservation", async(request, response) => {
+  const id = Math.floor(Math.random() * 2);
+  response.json(reservations[id]);
+});
+
+
+app.listen(3000, function () {
```
Comment (qamarfj) [inline nodejs/week1/homework/src/backend/app.js]: Since we have this code in server.js file therefore here we will only export app module.

**Review by qamarfj — COMMENTED** [review]
Hi @varsha84 ,
You have done good work,  some tests fail when we run "npm test"
Test Suites: 1 failed, 1 total
Tests:       3 failed, 4 passed, 7 total.
You code should pass all the tests.

File: nodejs/week1/homework/src/backend/app.js (hunk 1 above)
Comment (varsha84) [reply nodejs/week1/homework/src/backend/app.js]: Thanks for your feedback. I found my mistake and also learn how to run test.

**Review by qamarfj — APPROVED** [review]
👍

**Pull request merged**